hackathon_site/registration/forms.py

```diff
@@ -11,6 +11,8 @@
 from registration.models import Application, Team, User
 from registration.widgets import MaterialFileInput
 
+import re
```

Comment:
Can you move this up to below the `dateutil` import, with the rest of the stdlib imports?

Reply:
Still need to move this to below the `dateutil` import to keep our imports organized

Reply:
Still needs to be moved up where it should be